src/routing.js

```diff
@@ -4,7 +4,7 @@
  * Edge class
  */
 class Edge {
-  constructor(destination, weight=1) {
+  constructor(destination, weight = 1) {
     this.destination = destination;
     this.weight = weight;
```

Comment:
I don't think you really needed the weight for this assignment, but good job on including it.